run/local-cluster.sh

```diff
@@ -139,6 +139,11 @@ fi
 # Fetch the local status contents from node 1 and verify build information is present.
 LOCAL_URL="$DOCKERHOST:${HTTP_PORTS[1]}/_status/local/"
 LOCAL=$(curl --noproxy '*' -s $LOCAL_URL)
+if [[ -z $LOCAL ]]; then
```

Comment:
It already fails if the container hasn't started yet, but previously it was giving the unhelpful message "Build var missing for goVersion in $LOCAL" with no log output. Now when it fails it will dump the logs from the node. 

The failures I was seeing were for a node that crashed at startup; if we have a problem with nodes that successfully start but take some time to come up then we could wrap this in a loop (like the gossip test below), but it hasn't been a problem yet.